src/tools/tidy/src/rustdoc_js.rs

```diff
@@ -9,9 +9,9 @@ use ignore::DirEntry;
 
 use crate::walk::walk_no_read;
 
-fn run_eslint(args: &[PathBuf], config_folder: PathBuf, bad: &mut bool) {
+fn run_eslint(eslint_version: &str, args: &[PathBuf], config_folder: PathBuf, bad: &mut bool) {
     let mut child = match Command::new("npx")
-        .arg("eslint")
+        .arg(format!("eslint@{eslint_version}"))
```

Comment:
@jieyouxu It should always try to use this version but I am not sure what the dependency resolution process for npx is from here, e.g. if it will use a compatible version that isn't the specified one.